### PR TITLE
Add some node pattern predicates examples

### PR DESCRIPTION
**Pull request by inmost-light**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala [hunk 1]
```diff
@@ -474,6 +477,34 @@ class WhereTest extends DocumentingTest {
           resultTable()
         }
       }
+      section("Pattern element predicates", "pattern-element-predicates") {
+        section("Node pattern predicates", "node-pattern-predicates") {
+          p("`WHERE` clause can appear inside a node pattern in `MATCH` clause or pattern comprehension:")
```
Comment (tobias-johansson) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala]:
```suggestion
          p("`WHERE` can appear inside a node pattern in a `MATCH` clause or a pattern comprehension:")
```

**Review by tobias-johansson — CHANGES_REQUESTED** [review]
Looks good! I just have one small spelling/grammar suggestion.

**Review by martin-neotech — APPROVED** [review]
This is great! Did not know that Cypher could do that.

**Comment by inmost-light** [comment]
@martin-neotech 
That's new in 4.4.

**Review by tobias-johansson — APPROVED** [review]

**Pull request merged**